service/matching/matchingEngine.go

```diff
@@ -934,6 +933,27 @@ func (e *matchingEngineImpl) ApplyTaskQueueUserDataReplicationEvent(
 	return &matchingservice.ApplyTaskQueueUserDataReplicationEventResponse{}, err
 }
 
+func (e *matchingEngineImpl) ForceUnloadTaskQueue(
```

Comment:
Should we assert that this isn't called unless some test dynamic config is called?

Reply:
I think it's fine. The equivalent function for history (CloseShard) has no special protection. It could be useful in some emergency operational scenario